react/components/Link.tsx

```diff
@@ -65,7 +67,8 @@ const Link: React.FunctionComponent<Props> = ({
         replace,
         modifiersOptions,
       }
-      if (navigate(options)) {
+
+      if ((target === '_self' || !target) && navigate(options)) {
```

Comment:
I would like the opinion of you guys about this, because if the user passes `target="_top"`  or `target="_parent"` it might behave in a way that we don't want

Reply:
comment this on the code